core/controller/src/main/scala/whisk/core/controller/RestAPIs.scala

```diff
@@ -195,6 +198,21 @@ protected[controller] class RestAPIVersion_v1()(
     private val webexp = new WebActionsApi(Seq("experimental", "web"), WebApiDirectives.exp)
     private val web = new WebActionsApi(Seq("web"), WebApiDirectives.web)
 
+    /**
+     * Describes details of a particular API path.
+     */
+    private val info = JsObject(
+        "description" -> "OpenWhisk API".toJson,
+        "min_cli_version" -> whiskConfig(whiskVersionDate).toJson,
+        "min_recommended_cli_version" -> whiskConfig(whiskVersionDate).toJson,
```

Comment:
Also, not sure I like having two info routes since they both seem to share some of the same information (build, api, path, etc).